libraries/AP_Math/fenv.h

```diff
@@ -0,0 +1,42 @@
+#if !defined(__APPLE__) && !defined(__MACH__)
```

Comment:
So if \_\_APPLE\_\_ **or** \_\_MACH\_\_ is defined the rest of this file is included. Why you use APPLE define here but not in the Util file?